xtask/src/run.rs

```diff
@@ -358,28 +357,8 @@ pub fn run(opts: Options) -> Result<()> {
                 if let Some(cpu) = cpu {
                     qemu.args(["-cpu", cpu]);
                 }
-                if guest_arch == ARCH {
-                    match OS {
-                        "linux" => {
-                            const KVM: &str = "/dev/kvm";
-                            match OpenOptions::new().read(true).write(true).open(KVM) {
-                                Ok(_file) => {
-                                    qemu.args(["-accel", "kvm"]);
-                                }
-                                Err(error) => match error.kind() {
-                                    ErrorKind::NotFound | ErrorKind::PermissionDenied => {}
-                                    _kind => {
-                                        return Err(error)
-                                            .with_context(|| format!("failed to open {KVM}"));
-                                    }
-                                },
-                            }
-                        }
-                        "macos" => {
-                            qemu.args(["-accel", "hvf"]);
-                        }
-                        os => bail!("unsupported OS: {os}"),
-                    }
+                for accel in ["kvm", "hvf", "tcg"] {
```

Comment:
Interesting that passing `kvm` works on macOS

Reply:
Yep it will just try them sequentially